lib/gris/setup.rb

```diff
@@ -19,5 +19,28 @@ def env=(environment)
     def db_connection_details
       YAML.load(ERB.new(File.read('./config/database.yml')).result)[Gris.env]
     end
+
+    def cache
+      @_cache ||= ActiveSupport::Cache.lookup_store(:memory_store)
+    end
+
+    def cache=(store_option)
+      @_cache = ActiveSupport::Cache.lookup_store(store_option)
+    end
+  end
+end
+
+# prevent DalliError: Unable to unmarshal value: undefined class/module
+Gris.cache.instance_eval do
+  def fetch(key, options = {}, rescue_and_require = true)
+    super(key, options)
+
+  rescue ArgumentError => ex
```

Comment:
If you are not in rails, unmarshaling an ActiveRecord object blows up with that error in the comment unless you use `require` or `require_dependency` in each controller that uses the cache.